Data/converted_ADM_stellartable.dat

```diff
@@ -0,0 +1,2001 @@
+# r mass_density temp Ye specific_internal_energy velocity pressure adm_density adm_momentum S_adm Srr_adm
```

Comment:
As we discussed, we should remove this file, and instead provide instructions for attaining a progenitor file and then generating the ADM version.